content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,246 @@
+branch is actually supported.
+
+```diagram
+   v1.0.0                                                          ^
+ Sep 2, 2020                                                       | UNSUPPORTED
+------+---------------------------------------------> release-1.0  | RELEASES
+       \                                                           v
+        \
+         \       v1.1.0
+          \    Nov 24, 2021                                        ^
+           ---------+-------------------------------> release-1.1  |
+                     \                                             | SUPPORTED
+                      \                                            | RELEASES
+                       \         v1.2.0                            | = the two
+                        \      Feb 10, 2021                        |   last
+                         ------------+--------------> release-1.2  |   releases
+                                      \                            v
+                                       \
+                                        \
+                                         \
+                                          -----------> master branch
+                                                       April 1, 2021
+```
+
+### Technical support {#technical-support}
+
+Technical assistance is offered on a best-effort basis for supported
+releases only. You can request support from the community on [Kubernetes
+Slack](https://slack.k8s.io/) (in the `#cert-manager` channel), using
+[GitHub Discussions][discussions] or using the [cert-manager-dev][group]
+Google group.
+
+[discussions]: https://github.com/jetstack/cert-manager/discussions
+[group]: https://groups.google.com/g/cert-manager-dev
+
+### Security and bug fixes {#bug-fixes-support}
+
+We back-port important bug fixes — including security fixes — to all
+currently supported releases.
+
+- [Security issues](#security-issues),
+- [Critical bugs](#critical-bugs),
+- [Long-standing bugs](#long-standing-bugs).
+
+#### Security issues {#security-issues}
+
+**Security issues** are fixed as soon as possible. They get back-ported to
+the last two releases, and a new patch release is immediately created for
+them.
+
+#### Critical bugs {#critical-bugs}
+
+**Critical bugs** include both regression bugs as well as upgrade bugs.
+
+Regressions are functionalities that worked in a previous release but no
+longer work. [#3393][] and [#2857][] are two examples of regressions.
+
+Upgrade bugs are issues (often Helm-related) preventing users from
+upgrading to currently supported releases from earlier releases of
+cert-manager. [#3882][] and [#3644][] are examples of upgrade bugs.
+
+Note that intentional breaking changes do not belong to that category.
+Intentional breaking changes are changes that have been announced at least
+four months prior. For example, an intentional breaking change that is
+shipped in release 1.4 will first be announced in 1.2.
```

Comment:
question:

1. is four months something we've discussed and agreed on? It seems an exceptionally long time for some changes.

2. is it worth giving intentional breaking changes their own section and referring to that section in the "critical bugs" section?

Reply:
> Give breaking changes their own section

Good point, I'll do that.

> is four months something we've discussed and agreed on? It seems an exceptionally long time for some changes.

I have to admit that we never agreed on anything regarding this "4 months notice period". I might have picked this 😅

But yeah, I do agree that it seems to be an extremely long notice period... but that's something I would really expect from cert-manager: don't break things too fast! 😇

Thoughts? @jakexks @irbekrm @JoshVanL @wallrj @munnerz

Reply:
Done in b6a580f 👍